src/Acquisition.jl

```diff
@@ -35,4 +37,43 @@ include("plot.jl")
 include("calc_powers.jl")
 include("est_signal_noise_power.jl")
 include("acquire.jl")
+
+
+@setup_workload begin
+    # Putting some things in `@setup_workload` instead of `@compile_workload` can reduce the size of the
+    # precompile file and potentially make loading faster.
+    rate = 2.048e6 
+
+    #common real data
+    data_i8 = rand(Int8,Int(round(rate*0.01)))
+    data_i16 = rand(Int16,Int(round(rate*0.01)))
+    data_f32 = rand(Float32,Int(round(rate*0.01)))
+    data_f64 = rand(Float64,Int(round(rate*0.01)))
+
+
+    #common complex data
+    data_ci8 = rand(Complex{Int8},Int(round(rate*0.01)))
+    data_ci16 = rand(Complex{Int16},Int(round(rate*0.01)))
+    data_cf32 = rand(Complex{Float32},Int(round(rate*0.01)))
+    data_cf64 = rand(Complex{Float64},Int(round(rate*0.01)))
```

Comment:
I think it is fine if we just hard code the number of samples to something like a `1000`:
```suggestion
    signal_types = (Int16, Float32, Float64)
    signals = map(T -> rand(Complex{T}, 1000), signal_types)
```
We probably also need different `sample_rates` types, `interm_freq` types and `max_doppler` types
```
samplerates = (10000Hz, 10000.0Hz, 1000u"kHz", 1000.0u"kHz", 10u"MHz", 10.0u"MHz")
interm_freqs = (100Hz, 100.0Hz, 100u"kHz", 100.0u"kHz", 1u"MHz", 1.0u"MHz")
max_dopplers = (1000Hz, 1000.0Hz, 1u"kHz", 1.0u"kHz")
```
The GPSL1 should be initialized before `@compile_workload` because it isn't part of Acquisition.jl .
```
systems = (GPSL1(), GPSL5(), GalileoE1B())
```